packages/blockchain-wallet-v4-frontend/src/scenes/Exchange/ExchangeInfo/index.tsx

```diff
@@ -0,0 +1,98 @@
+import { actions, selectors } from 'data'
+import { Box, Container } from 'components/Box'
+import { connect, ConnectedProps } from 'react-redux'
+import { Dispatch } from 'redux'
+import { FormattedHTMLMessage, FormattedMessage } from 'react-intl'
+import { Icon, Text } from 'blockchain-info-components'
+import { SuccessCartridge } from 'components/Cartridge'
+import React from 'react'
+import styled from 'styled-components'
+
+const ContainerWithBackground = styled(Container)`
+  background-image: url('/img/swap-info.png');
+  /* stylelint-disable */
+  background-image: -webkit-image-set(
+    url('/img/swap-info.png') 1x,
+    url('/img/swap-info@2x.png') 2x
+  );
+  /* stylelint-enable */
+  background-repeat: no-repeat;
+  border-radius: 8px 8px 0 0;
+  box-sizing: border-box;
+  max-width: 328px;
+  background-size: 333px 129px;
+  margin-left: 24px;
+`
+
+const Header = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin-bottom: 20px;
+`
+
+const ExchangeInfo = ({ showSwapInfoBanner, hideSwapInfoBanner }) => {
+  if (!showSwapInfoBanner) {
+    return null
+  }
+  return (
+    <ContainerWithBackground>
+      <Box>
+        <Header>
+          <SuccessCartridge style={{ textTransform: 'uppercase' }}>
+            <FormattedMessage id='copy.new' defaultMessage='New' />
+          </SuccessCartridge>
+          <Icon
+            cursor
+            name='close-circle'
+            size='20px'
+            color='grey400'
+            onClick={() => hideSwapInfoBanner()}
+          />
+        </Header>
+        <div>
+          <Text
+            size='20px'
+            weight={600}
+            color='grey900'
+            style={{ marginBottom: '10px' }}
+          >
+            <FormattedHTMLMessage
+              id='swap.getstarted.readytoswap.info.title'
+              defaultMessage='Swap is Now<br />Faster & Cheaper.'
```

Comment:
thoughts on HTML in the string @plondon?  I cant remember if we want to support this or not.  I know we have had issues with the translators before not understanding the meaning of this.

Reply:
@schnogz didn't been aware of this, however design here is a bit odd and we need to put it in next line on that place and usage of `\n` didn't worked well.